code/datums/blood_type.dm

```diff
@@ -0,0 +1,70 @@
+/datum/blood_type
+	/// Displayed name of the blood type.
+	var/name = "?"
+	/// Shown color of the blood type.
+	var/color = COLOR_BLOOD
+	/// Blood types that are safe to use with people that have this blood type.
+	var/compatible_types = list()
+
+/datum/blood_type/a_minus
+	name = "A-"
+	compatible_types = list(/datum/blood_type/a_minus, /datum/blood_type/o_minus)
+
+/datum/blood_type/a_plus
+	name = "A+"
+	compatible_types = list(/datum/blood_type/a_minus, /datum/blood_type/a_plus, /datum/blood_type/o_minus, /datum/blood_type/o_plus)
+
+/datum/blood_type/b_minus
+	name = "B-"
+	compatible_types = list(/datum/blood_type/b_minus, /datum/blood_type/o_minus)
+
+/datum/blood_type/b_plus
+	name = "B+"
+	compatible_types = list(/datum/blood_type/b_minus, /datum/blood_type/b_plus, /datum/blood_type/o_minus, /datum/blood_type/o_plus)
+
+/datum/blood_type/ab_minus
+	name = "AB-"
+	compatible_types = list(/datum/blood_type/b_minus, /datum/blood_type/a_minus, /datum/blood_type/ab_minus, /datum/blood_type/o_minus)
+
+/datum/blood_type/ab_plus
+	name = "AB+"
+	compatible_types = list(/datum/blood_type/b_minus, /datum/blood_type/a_minus, /datum/blood_type/ab_minus, /datum/blood_type/o_minus)
```

Comment:
Wha

Reply:
\> Be me, first day as AB+ blood bank supervisor
\> AB+ blood accepts any other kind of human blood, from O- to AB+
\> Look inside AB+ blood bank
\> It's all AB-
(I may be going insane but I don't think this is right?)

Reply:
AB+ is an universal recipient, which includes AB- (but not the other way around)

Reply:
Look at the code my dearest chap

Reply:
IDK this is how it was